cmd/dynamic_startup_test.go

```diff
@@ -159,50 +153,4 @@ func TestGetSnapshotAtEpochAndPhase(t *testing.T) {
 		require.Equalf(t, 1, hookInfoCalls, "expected 1 info log got %d", hookInfoCalls)
 		require.Equal(t, expectedSnapshot, actualSnapshot)
 	})
-	t.Run("should wait for target phase if target epoch has passed but not in target phase", func(t *testing.T) {
```

Comment:
It would be good to test the typical case. If a node is joining at epoch 5 and they run with the default configs (epoch_counter=0, epoch_phase=setup), they should not start up during staking phase of epoch 5, but should start up during the setup phase of epoch 5.

Reply:
Should a node start up immediately if the target epoch has passed ?

Reply:
Yes, definitely. It should always start up as soon as possible after it is able to.

Reply:
https://github.com/onflow/flow-go/pull/1893/commits/a24cc2a5241a68045645e4d1203580e1f759b6ae